morph_tool/spatial.py

```diff
@@ -27,3 +34,47 @@ def point_to_section_segment(neuron, point):
             return (section.id + 1, offset[0][0])
 
     raise ValueError('Cannot find point in morphology that matches: {}'.format(point))
+
+
+def align_pyr_morph(morph, direction=None):
+    """In place alignment of a pyramidal morphology such that apical tree is along 'direction'.
+
+    The algorithm is based on SVD decomposititon of the correlation matrix obtained from all points
+    in apical neurites, giving the principal axis of apical dendrite.
+    If not apicals are present, no rotation is applied.
```

Comment:
Shouln't it be `If no apical is present`?

Reply:
yes